std/format.d

```diff
@@ -1735,10 +1735,22 @@ FormatSpec!Char singleSpec(Char)(Char[] fmt)
  * `toString` should have one of the following signatures:
  *
  * ---
- * const void toString(scope void delegate(const(char)[]) sink, FormatSpec fmt);
- * const void toString(scope void delegate(const(char)[]) sink, string fmt);
- * const void toString(scope void delegate(const(char)[]) sink);
- * const string toString();
+ * void toString(W)(W w, FormatSpec fmt)
+ * void toString(W)(W w)
+ * string toString();
+ * ---
+ *
+ * Where `W` allows a character accepting
```

Comment:
What about "Where W is an output range that accepts ...".

Reply:
Also, it really should be `ref W`. Accepting an output range by value is begging for trouble.